docker/production/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  edpn-frontend-docker-production:
```

Comment:
To make Ops life easier container names should probably follow a convention using the `container_name` field.

Not a priority right now, and probably solved by WGLs agreeing on something. So far, most services running on Morris have gone for _edpn-<service>-n_ - but this can be changed easily.

Reply:
This should be `edpn-<service>-n`